examples/example_config_files/isochrone_test.json

```diff
@@ -0,0 +1,133 @@
+{
+    "name": "Isochrone Bug",
```

Comment:
The `"Isochrone Bug"` detector also has a weird geometry, the contacts are reaching into the bulk of the detector at the bottom of the detector.. I'm wondering why the tests do not fail for points in the region shielded from the point contact by the mantle contact 

![Isochrone Test Detector](https://user-images.githubusercontent.com/30291312/173334523-a44e8087-d8ec-4a7a-a617-2c28e8d30c45.png)

Reply:
Ok, I guess it's because you only select part of `DT` in the tests and only look at the hole drift paths, not the electrons.

```julia
using SolidStateDetectors
using Unitful
using Test
using Plots; gr(fmt = :png)
T = Float32

sim = Simulation{T}(SSD_examples[:IsochroneTest])
simulate!(sim)
calculate_electric_potential!( sim, refinement_limits = [0.2, 0.1, 0.05, 0.01])
calculate_electric_field!(sim, n_points_in_φ = 36)
charge_drift_model = ADLChargeDriftModel()
sim.detector = SolidStateDetector(sim.detector, charge_drift_model)

spawn_positions = CartesianPoint{T}[]
idx_spawn_positions = []
x_axis = 0:0.0005:0.04
z_axis = -0.005:0.0005:0.045
spawn_positions = CartesianPoint{T}[]
idx_spawn_positions = CartesianIndex[]
for (i,x) in enumerate(x_axis)
    for (k,z) in enumerate(z_axis)
        push!(spawn_positions, CartesianPoint(T[x,0,z]))
        push!(idx_spawn_positions, CartesianIndex(i,k))
    end
end

in_idx = findall(x -> x in sim.detector && !in(x, sim.detector.contacts), spawn_positions);
length(in_idx)

ev = Event(spawn_positions[in_idx]);
time_step = T(1)u"ns"
max_nsteps = 10000
drift_charges!(ev, sim, Δt = time_step, max_nsteps = max_nsteps, verbose = false)

DT_h = Array{typeof(time_step),2}(fill(typeof(time_step)(NaN),length(x_axis),length(z_axis)))
DT_e = Array{typeof(time_step),2}(fill(typeof(time_step)(NaN),length(x_axis),length(z_axis)))
for (i, idx) in enumerate(idx_spawn_positions[in_idx])
    DT_h[idx] = length(ev.drift_paths[i].h_path)*time_step
    DT_e[idx] = length(ev.drift_paths[i].e_path)*time_step
end
```

```julia
# Holes
contourf(vcat(-x_axis[end:-1:2], x_axis)*1000u"mm", z_axis*1000u"mm", DT_h[vcat(end:-1:2,1:end),:]', 
    ratio = 1, 
    size = (500,500), colorbar_title = "Driftime", unitformat = :slash, 
    xlabel = "x", ylabel = "z", clims = (0,Inf))
```
![Holes](https://user-images.githubusercontent.com/30291312/173335710-146e4ed0-2d4d-413c-9a39-1649e7929245.png)


```julia
# Electrons
contourf(vcat(-x_axis[end:-1:2], x_axis)*1000u"mm", z_axis*1000u"mm", DT_e[vcat(end:-1:2,1:end),:]', 
    ratio = 1, 
    size = (500,500), colorbar_title = "Driftime", unitformat = :slash, 
    xlabel = "x", ylabel = "z", clims = (0,Inf))
```
![Electrons](https://user-images.githubusercontent.com/30291312/173335716-f6b2037f-2aa9-4111-aed4-ca0a206658f0.png)